assets/src/edit-story/components/library/libraryProvider.js

```diff
@@ -28,29 +28,36 @@ import { useState } from '@wordpress/element';
  * Internal dependencies
  */
 import Context from './context';
+import useInsertElement from './useInsertElement';
```

Comment:
FYI for drag and drop into page, it needs to insert into page. I want to move this the `insertElement` method to somewhere easier to get at global. Can we do this as part of this refactor as well.

Reply:
Hm, it doesn't actually do a lot more than `addElement` from the global story provider besides capping the width and height and converting media stuff to background elements. But it does do some logic, so it's probably a valid request. I wouldn't really know where to place it globally even.

Alternatively, drag-n-drop could just use the same `useInsertElement` hook and we could move that to a more general location?

Reply:
For reference. https://github.com/google/web-stories-wp/pull/376